spec.html

```diff
@@ -14349,7 +14350,7 @@ <h2>Syntax</h2>
 
       <emu-clause id="sec-static-semantics-sv" oldids="sec-string-literals-static-semantics-stringvalue" type="sdo" aoid="SV">
         <h1>Static Semantics: SV</h1>
-        <p>A string literal stands for a value of the String type. The String value (SV) of the literal is described in terms of String values contributed by the various parts of the string literal. As part of this process, some Unicode code points within the string literal are interpreted as having a mathematical value (MV), as described below or in <emu-xref href="#sec-literals-numeric-literals"></emu-xref>.</p>
+        <p>A string literal stands for a value of the String type. The SV of the literal is described in terms of String values contributed by the various parts of the string literal. As part of this process, some Unicode code points within the string literal are interpreted as having a mathematical value, as described below or in <emu-xref href="#sec-literals-numeric-literals"></emu-xref>.</p>
```

Comment:
Here with SV and below with TV and TRV, how come you removed the expansion of the acronyms? I feel like they read a bit clearer with the expansion.